_guide/_pages/index.md

```diff
@@ -15,7 +15,7 @@ This guide is where the TTS Engineering Practices Guild collects its best practi
 
 ## How we classify best practices
 
-These documents are structured by topic; under each, we include "Requirement",
+These documents are structured by topic; under several topics we indicate "Requirement",
 "Standard", "Default", "Suggestion", and "Caution".
```

Comment:
```suggestion
"Standard", "Default", "Suggestion", and "Caution".

If topics are not classified, the decision is left at your discretion. If you are unsure, ask in #dev, as the topic may be a good candidate for classification.
```

Reply:
Suggest moving the absence of a classification up, and to indicate what to do if you'd like more guidance.